docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking the DeepCell Kiosk
+---------------------------------------------------------
+
+The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To run your own benchmarking, please read below.
+
+1. If you don't already have a cloud storage bucket devoted exclusively to DeepCell Kiosk benchmarking runs, you should create one now. It's fine to reuse this bucket for multiple benchmarking runs, but don't use it as the storage bucket for normal DeepCell Kiosk cluster operations.
+
+2. Then, checkout the ``benchmarks`` branch of the ``kiosk-console`` repository. This branch is almost identical to the version 1.2.0 release of the ``kiosk-console`` repo, except for minor configuration differences. There are three variable settings worth noting in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, each with a corresponding setup action for the user to perform before benchmarking:
```

Comment:
I'm not sure if we should recommend a `git checkout` but maybe just use it as a reference for our own configuration?

I mean to say that the `benchmarks` branch is the configuration we used, but they can just use it as a reference rather than needing to start from that branch. None of those settings is _required_ to be changed, but they just show what the settings we used are.

Reply:
Oh, so you want to tell people they can run benchamarking off master? Can we do that in the figure recreation section, too?

Reply:
Yeah exactly, the benchmarking pod will work fine from `master`. The `benchmarks` branch just shows the configuration settings we used and sets them as default choices during the cluster config menu.

For figure recreation, I would just point them to the benchmarking branch as an example of all the required changes. Maybe still walk them through each change for explicit-ness.